hubmap_commons/globus_groups.py

```diff
@@ -5,7 +5,8 @@
 
 # HuBMAP commons
 from hubmap_commons import string_helper
-from hubmap_commons.hm_auth import AuthHelper, AuthCache
+from hubmap_commons.hm_auth import AuthHelper
+from hm_auth import AuthCache
```

Comment:
@maxsibilla I tested with using your commons branch locally as a dependency in entity-api, and got this error:
```
    from hubmap_commons import globus_groups
  File "/usr/local/lib/python3.6/site-packages/hubmap_commons/globus_groups.py", line 9, in <module>
    from hm_auth import AuthCache
ModuleNotFoundError: No module named 'hm_auth'
```
You'll need to either keep the old import 
```
from hubmap_commons.hm_auth import AuthHelper, AuthCache
```
or import separately as 
```
from hubmap_commons.hm_auth import AuthCache
```

Reply:
@yuanzhou Since I am modifying hm_auth I couldn't do that library import as that would pull the code from the 2.0.13 version of the file without my changes. I modified my setup in PyCharm to support that local module so that might explain why you are getting that error. I will look to see how to fix that

Reply:
@maxsibilla for local testing, you can still install the commons of your branch from github when you install this package as a dependency. For example, in entity-api:
```
export COMMONS_BRANCH=globus-groups-changes
pip install -r requirements.txt
```
Afterwards, when you  fire up the entity-api with Flask dev server, the logging will tell you if anything wrong with the import. Sometimes IDE like Eclipses or PyCharm may complain about import errors, but when you run from command line with using IDE, it works. Our docker build does the build like above.

Reply:
Thanks, I modified the import statement